src/cascade/executor/create_settings.py

```diff
@@ -0,0 +1,250 @@
+        "fixed": 1e-10
+    }
+}
+
+
+def grid(non_negative, rng, single_age=False):
+    grid_case = {
+        "default": {
+            "value": {
+                "density": "gaussian",
+                "mean": 0.1,
+                "std": 0.1
+            },
+            "dage": {
+                "density": "gaussian",
+                "mean": 0,
+                "std": 0.1
+            },
+            "dtime": {
+                "density": "gaussian",
+                "mean": 0,
+                "std": 0.1
+            }
+        },
+        "age_time_specific": rng.choice([0, 1], p=[0.25, 0.75]),
+    }
+    if non_negative:
+        # The min has to be > 0?
+        grid_case["default"]["value"]["min"] = rng.choice([1e-4])
+        grid_case["default"]["value"]["mean"] = grid_case["default"]["value"]["min"] + 1e-2
+    age_choice = rng.choice([0, 1, 2])
+    if age_choice == 1:
+        grid_case["age_grid"] = "0"
+    elif age_choice == 2:
+        grid_case["age_grid"] = "0 1 5 10"
+    if single_age:
+        grid_case["age_grid"] = "0"
+    time_choice = rng.choice([0, 1, 2])
+    if time_choice == 1:
+        grid_case["time_grid"] = "2000"
+    elif time_choice == 2:
+        grid_case["time_grid"] = "2000 2005 2010"
+    return grid_case
+
+
+def rate_grid(rate, rng):
+    single_age = rate == "pini"
+    grid_case = grid(True, rng, single_age)
+    grid_case["rate"] = rate
+    return grid_case
+
+
+def covariate(study_country, covariate_idx, rng):
+    assert study_country in ["study", "country"]
+    grid_case = grid(False, rng)
+    grid_case["mulcov_type"] = rng.choice(["meas_std", "meas_value", "rate_value"])
+    grid_case["measure_id"] = 41
+    grid_case[f"{study_country}_covariate_id"] = covariate_idx
+    grid_case["transformation"] = 0
+    return grid_case
+
+
+def create_settings(rng):
+    """
+    Makes a random settings, as though EpiViz-AT made it.
+
+    Args:
+        rng (numpy.random.RandomState): Create with ``RandomState(2324234)``.
+
+    Returns:
+        dict: A dictionary that looks like parsed JSON from EpiViz-AT.
+    """
+    has = dict()
+    for rate, likely in [("iota", 0.1), ("rho", 0.7), ("omega", 0), ("chi", 0.1), ("pini", 0.7)]:
+        has[rate] = rng.uniform() > likely
+
+    # The parent location id is 1
+    # Would you ever have one child random effect?
+    child_selection = [None, [2], [2, 3, 4]]
+    children = child_selection[rng.choice([0, 1, 2], p=[0.2, 0.05, 0.75])]
+
+    covariates = [1604, 2453, 6497]
+
+    case = deepcopy(BASE_CASE)
+    if has["pini"]:
+        case["model"]["birth_prev"] = 1
+
+    case["model"]["add_calc_emr"] = rng.choice([0, 1])
+    case["model"]["constrain_omega"] = rng.choice([0, 1], p=[0.1, 0.9])
+
+    # Guarantee at least one nonzero rate, after constraint of omega.
+    if case["model"]["constrain_omega"] == 1:
+        has["omega"] = True
+        if not any(has[x] for x in ["iota", "rho", "chi"]):
+            has["iota"] = True
```

Comment:
I needed to make one there alone if no others are there. If there is one other one there (which could be any other) then we're fine. I could choose numbers between 1 and 32, excluding 0, and use bits... This seems like it will hit all cases.